app/views/organizations/staff/adoption_application_reviews/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= render DashboardPageComponent.new(crumb: :dashboard_applications, crumb_options: [@application]) do |c| %>
+  <% c.with_header_title do %>
+    Answers by <%= @application.person.full_name %> <br/>
+    <%= @application.person.email %>
+  <% end %>
+  <% c.with_body do %>
+     <!-- row -->
+    <% @form_answers.each do |form_answer| %>
+      <div class="justify-content-md-between mb-4 mb-xl-0 gx-3">
+      <!-- card -->
+      <div class="card">
+        <div id="<%= dom_id form_answer %>"class="card-body d-flex flex-sm-row flex-column justify-content-between border-bottom">
+          <div class="d-flex align-items-center">
+            <div>
+              <strong class="fs-4" >Q: <%= form_answer.question_snapshot %></strong>
+              <p class="">A: <%= form_answer.value %> </p>
```

Comment:
We can get rid of the margin on the bottom of this `<p>`
<img width="367" alt="image" src="https://github.com/user-attachments/assets/a841c08a-59d6-4362-9739-bf9710176d58">

Reply:
Done